cert/aziot-certd/src/lib.rs

```diff
@@ -64,6 +64,14 @@ pub async fn main(
         principal,
     } = config;
 
+    #[cfg(ossl330)]
+    {
+        let ossl_providers = openssl::provider::Provider::load(None, "default");
```

Comment:
I assume this is needed because:

>https://www.openssl.org/docs/man3.0/man7/OSSL_PROVIDER-default.html
>
>If an application doesn't specify anything else explicitly (e.g. in the application or via config), then this is the provider that will be used as fallback: It is loaded automatically the first time that an algorithm is fetched from a provider or a function acting on providers is called and no other provider has been loaded yet.
>
>If an attempt to load a provider has already been made (whether successful or not) then the default provider won't be loaded automatically.

... ?

Reply:
Yes